app/Http/Controllers/FaqController.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Faq;
+use Illuminate\Http\Request;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\View\View;
+
+class FaqController extends Controller
+{
+    /**
+     * Display the index FAQ page.
+     *
+     * @return View The faqs/index.blade.php view
+     */
+    public function index(): View
+    {
+        return view('faqs.index', [
+            'faqs' => Faq::simplePaginate(10)
+        ]);
+    }
+
+    /**
+     * Create a new FAQ
+     * @return View The faqs/create.blade.php view
+     */
+    public function create(): View
+    {
+        return view('faqs.create');
+    }
+
+    /**
+     * Store a new FAQ
+     * @param Request $request
+     * @return RedirectResponse
+     */
+    public function store(Request $request): RedirectResponse
+    {
+        $validated = $request->validate([
+            'question' => 'required|string|min:10|max:255',
+            'link' => 'nullable|url',
+            'answer' => ['required', 'string', function ($attribute, $value, $fail) {
+                if (strlen(strip_tags($value)) < 10) {
```

Comment:
It might be more convenient to just use the "min:10" validation rule instead. Stripping tags before checking does not add much to the users experience.